src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1293,14 +1293,15 @@ public boolean unsolvedAndNotSimple(MethodCallExpr method) {
   }
 
   /**
-   * For a method call that is not simple, this method will take that method as input and create
-   * corresponding synthetic class.
+   * Creates a synthetic class corresponding to a static method called by a qualified class name.
+   * Ensure to check with {@link #isAnUnsolvedStaticMethodCalledByAQualifiedClassName} before
+   * calling this method.
    *
-   * @param methodCall the method call to be taken as input
-   * @param methodAgruments the list of agruments for this method call
+   * @param methodCall the method call to be used as input
```

Comment:
I would still prefer that the `@param` Javadoc here explains exactly the requirements on `methodCall` in string terms.

Reply:
That is, "must contain one or more dot separated identifiers, followed by a single pair of parentheses containing arguments" or something similar